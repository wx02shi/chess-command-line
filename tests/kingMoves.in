game human human
setup
+ K e1 
= black
+ K e8
done
move e1 e2
move e8 e7
move e2 d3
move e7 f6
move d3 e4
move f6 g7
move e4 d5
move g7 f7
move d5 c5
