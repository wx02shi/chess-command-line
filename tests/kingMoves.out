8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh
Setup completed!
8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ k _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ k _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _K_ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ k _
5 _ _ _ _ 
4  _ _ _ _
3 _ _K_ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ _ 
6  _ _ k _
5 _ _ _ _ 
4  _ _K_ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ k 
6  _ _ _ _
5 _ _ _ _ 
4  _ _K_ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _ k 
6  _ _ _ _
5 _ _K_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _k_ 
6  _ _ _ _
5 _ _K_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _
7 _ _ _k_ 
6  _ _ _ _
5 _ K _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
Final Score:
White: 0
Black: 0
